### PR TITLE
Gallery page

### PR DESCRIPTION
**Pull request by LenciCruz**
# Gallery Page

## Details

- Created second section + gallery page grid component of pictures only

## Juxtaposition
- 
image

- 
image_2024-07-18_172346161

- 
image

## Additional Notes

- page is not responsive
- video section still missing
- wave on images missing
- No footer

### CODE REVIEW

File: src/assets/images/gallery/1.png
Comment (JoshuaEntrata) [inline src/assets/images/gallery/1.png:1]: lipat mo lang yung pictures sa root folder `public/images/gallery/`

Comment (LenciCruz) [reply src/assets/images/gallery/1.png:1]: okay naa

File: src/components/Gallery/gallery-grid.tsx
Comment (JoshuaEntrata) [inline src/components/Gallery/gallery-grid.tsx:1]: pwede pala palagyan din ng `index.tsx` file itong `components/Gallery/` folder? tapos import export mo yung mga components sa folder na ito. After nun, i-update mo yung pag import dun sa `page.tsx` file. Pwede ka mag refer dun sa PR ko kung pano ko ginawa

Comment (LenciCruz) [reply src/components/Gallery/gallery-grid.tsx:1]: okay na rinnn thank you bossing

**Review by JoshuaEntrata — APPROVED** [review]

**Review by carlmitzchel — APPROVED** [review]

**Pull request merged**